src/seedu/addressbook/storage/jaxb/AdaptedPerson.java

```diff
@@ -60,7 +60,7 @@ public AdaptedPerson(ReadOnlyPerson source) {
 
         address = new AdaptedContactDetail();
         address.isPrivate = source.getAddress().isPrivate();
-        address.value = source.getAddress().value;
+        address.value = source.getAddress().toString();
```

Comment:
Good job in updating the unit tests.

Note that you should add tests at the system level (i.e. the input.txt and actual.txt).